src/Features/CSharp/Portable/ConvertNamespace/ConvertNamespaceTransform.cs

```diff
@@ -87,15 +88,12 @@ public static async Task<(Document document, TextSpan semicolonSpan)> ConvertNam
             if (openBraceLine == closeBraceLine)
                 return null;
 
-            var options = await document.GetOptionsAsync(cancellationToken).ConfigureAwait(false);
-            var style = options.GetOption(FormattingOptions.SmartIndent, document.Project.Language);
-
-            var indentation = indentationService.GetIndentation(document, openBraceLine + 1, style, cancellationToken);
+            // Auto-formatting options are not relevant since they only control behavior on typing.
+            var indentationOptions = new IndentationOptions(options, AutoFormattingOptions.Default);
```

Comment:
Unfortunately it can't. It calls helpers that read auto-formatting options. I agree it would certainly be better to separate those code paths but that is beyond the scope of this PR. Filed an issue here: https://github.com/dotnet/roslyn/issues/60178